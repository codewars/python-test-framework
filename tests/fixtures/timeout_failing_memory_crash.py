import codewars_test as test


from functools import lru_cache
from sys import setrecursionlimit
setrecursionlimit(54000)

Q = {*"nh|rh|ag|pa|b|h|pu|tb|ru|ts|cs|ir|no|cd|ne|og|np|hs|cu|pt|th|cr|ho|tm|lu|fl|ds|n|rn|k|sc|pm|ti|as|au|fe|pb|rg|s|u|ar|lv|at|hf|w|bh|in|na|tl|mt|v|xe|tc|cn|rb|sr|i|sm|br|ce|os|er|bi|eu|sb|sn|zr|sg|be|p|ni|ac|nb|rf|o|cl|cf|he|db|ra|ge|ga|nd|pr|mn|fr|f|pd|md|am|fm|ca|kr|lr|gd|mo|mc|mg|ta|se|y|bk|yb|si|es|re|co|po|li|te|c|hg|cm|al|zn|dy|la|ba".split('|')}
def XXX(s):
    @lru_cache(maxsize=None)
    def recurse(ind):
        S = 0
        for i in range(2):
            W = s[ind] if i == 0 else s[ind] + s[ind + 1]
            if W in Q:
                S += recurse(i + ind + 1)
        return S
    return recurse(0)



DATA = "sbinbhscsinbhosmosbinosbinposbhosbinbhsbinhosnhosinbinacnosinosmoscosbinpbhsmcnosinhosnosinosininhsinpbinosinpbhosninhsnosbhsbhscscosinhoscsbinposnpbhsinbhosmnbhscscninbinbinpbinhosnpbinposcninposninhosnosmcsnhsbhscosbhosnposcsmcscninposinirnininbhsinhosbininpbhsbhscoscsbhosmnbhoscsmcscoscsnhsnbhsnosbhosinosmnbinposbininhosininosbinpbininosinirbhsbhosmnpbinhosnhscnhsbhscosmnosinhsinhosnhsbhsnpbinascsnpbhsnhsbirhosbhoscscnbhosnbinosnhosnposbhosinpbinosinoscsninosbhsbinposinhsbhoscscnbhscnposcsmnosmnposnosmcsinposnpbhosnposnbhsnbhsbhsnposcnpbinpbhosmosnoscnbhsnpbhosnasbhsnpbhscnpbinposcnhosinbinhosbhscosbhsbhsnbinosnbhosnosnhoscnacosnpbinosninposnininosmnirhsnpbhsninhscnosbhosbhoscnasinbinhscscnbinosbhsbinhsnhsmcnhsinacnposbhosnbhsnhoscnacsnbhsinpbinposbhsbinhsbhsbhsininpbhsnhosbinhsbinpbhoscscoscsinhsnosnasbhosnpbhosbinposmnhosinpbhscnhsninpbhscnosninbinosbhscnhosininbinpbhsbhsinpbhsirhscsnbininpbinhosnposbinbhoscosnpbhscsinhsinhsmcsnasinposcoscnosininhsnosbinhsnhosmcnirbhosmosbinposnposnbinoscsininposinbhsbhosinpbinbhsnpbhsbhscosnpbhosbhosbhoscsinhscscoscosbhsbhsinascnhosnosbinposbhsnosmninosnposmoscosbinbhscnbhosinhsbhoscnbinbinosbhsnosnposbhosnbininbhsbhsnbhscosbhosbirbhsbinosbhscninhsbhosnosinposnhsnacnasbinbinbinhosinhscsmnpbhsbinposnposninbinpbhsbhsnininoscosnpbinbhoscsbirnosbhosnosnininacsbhsirnposmcnhoscscnacosbhosbhscninpbhoscnosnosbhsbinbhosnpbininbhosinosinbinposinhscnbhsmosirnbinpbhsninposcsnbhscsbhosnposinpbhsinhsbhosbinosirnosnpbhoscsnbhsmcosnpbinpbhosnbhsnhosbhscosnpbhscosbhsbinhsinhsinbinbinirnbhsinirbhsinpbhsbinhscsnpbhosbinbhscsninposininbhoscsbinposinpbinosnpbhosinosinpbhsinoscnpbirbinposcscnpbhosnbinhsninasinposnacosbhosmninposinpbhoscsinacosininpbinoscoscsbhsbhsnasnbhsnpbhosinhscnbhosnhsbhsbhsbhscscnhsinbhsmosbhosininbhoscsirhscsbinposcnbinposcnpbhsbinposinascsnbhsnposbhosinpbininhoscosnpbhoscnhsnposininhosnacsninbinoscsbinhsinhosnbhosbinhscnosnoscoscnposbinosninbhsninbinpbhosinhsinosnosnbinbininposnbinhosbhosbhosininpbhoscosninirnhsinosnpbinosinpbhosbhosninhoscsninposcsinasinbhosnposinosnhoscnpbinininosnpbhsnbhsnosinoscsbinpbinposnpbinpbhosinbhscosnininposnininposcnasbinosninhsinhsbinhsnposcnposbinbhsinhoscosinbinposnposininasbhsnbhsbinbhosnposbinpbhscsnpbhsinbhosinosmnosbhosbhsnininosbinbhsnposbinposnhosinbinposirnbinasinbinposnbhsmcosininosnbhsbinposnbinpbinosmninbinposbhsnoscnacnpbhosnbininhsmninpbinosmoscosbhosbinbinoscsmninbinposcscnbhosbinhosbhosinbhsnbhsnbhosnhoscnasbhsnhsininpbhsninhoscnbhosmcscsninoscninbinoscosbhsnbhosnosbhsninpbinposnbhsinhosinhscoscsmninhsbhsnbhsnacnpbininbinosbininbhsmcsnosbhoscnhosnosinacnbhsnhosinbinhsnpbininhsbhosinosbhosbhosbhosnhoscnacosininbhsninbinhosnhosnposcsinacnposnbinhsbirnposninbinhsninpbinpbhsbhosnacsbhosinhsbinpbhoscsbinposnirnpbhsmnpbinbinhsnoscosnosirbirnbhosnosnbinposcosninbhosnbhscsbhosinbinosnbhscsnbinbinposcosinosnposinbhsbinhscosbinhsinhsnosbhsbinbinosnosninininposmcninpbinpbhsinposnposcnpbinpbhsnbininbhsbhscnposbhscosnpbinhsnacninposnirbininininposcosnpbhscnhsninbhscosnposbhosinpbinbinbinbininposinposbininirhsnpbhsnoscnacsnosinininosninposmcoscnhscnininosinpbhosnosmnosbhscsnhosnbhoscsbhsmcscsmnbinposbinpbinbinhoscosnhsnosinoscnpbinpbhsnposbinbhosnpbinpbhsinhscsbhosbhscosininpbhsnpbhscnosnpbininpbhsbhosininbhosbhosinbhsnbhosnhscsbininposnhscosnbinposnposcsnhsnosinposcnbinpbhscnbinposcnbhsmosinpbinbhsninpbhsnbirbhsnosnbhsininhosnosinbhsninoscnposcosmnirnhosinosinposnosininhscnosnbinhsinhscsnpbhoscnosbinbhsnposmnbhsmnpbhosnpbhsbinpbhsinbhosnbinhsinhsnposbinbhosinpbinirbinpbininposirbhsinosnosnosinpbinhsnbinoscnasbhosbhosinbhsnbhsbhsmnbhosnhscosnhosbhsnbhosnhsnhsinosinbhscnhsnhsmnhscnbinbininhscsmnbinbinirnposnhsbinhscnbinposnbhscsbhosnpbinbinhscnosbinhosnpbhosinpbhsinosbinosnposinpbinosnposnpbinhosnosbinbhscnininbhsnpbhsinbhoscsinhsinbinhsnbirnbhoscsmnhsbhscsbhsininosinininpbinhsmcscnhoscnbhosinpbininoscnbhsninhsbhscosinosnhsnoscscnpbhscsinosninpbinhscnbhsinbhsbhsbinbinhsininbinhscnosbhsmnosnposmcsbinposbinininbhsnhsbhosinirbhosbinposcnosinosbhoscninoscnasbhscoscnpbhosbinpbinininininposcninhsnpbinininhsnbinhsbhsbinosinposnhsinosbinposinpbinbinpbhsnhsbinpbhscnposbhosnhosbhosbinpbhosinpbinosininosinbhsnpbinhsinbhosbhsnhscosinhosbinpbinbinposnhosmosinhsbhsbininhosnasbhsnbhscoscnasnpbhoscsnposnbinosnhosbirbhsnhosinpbhscsbinposinhscnosnposbhsirnbinposninhscsnbirbhsbhoscnosbhsnosinpbinhsininbinhosninposnposnhsninpbinbinininosnbirnbinhosinhsbinoscsnhscscnasninpbhsbhsinpbininosbhsinposcnosinoscnosmcosnposcoscnpbhsnbinpbhsbinbinhosnpbhoscsbhsnhoscosnbinhsbhosbhosbhsinbinbinhosmninoscscscnposnpbhsnbinosnhosbininbininbhosbinhsnhsbinbhsnbhsbhosmcnosnhosbirhsinoscsnposcninhscnpbhsnosbhosnpbininhscscnpbhoscoscsnposininhsnosbhsnposbhscosbhosbhosnhsmcnhsbhscnbinoscnpbinosbhosininhscsninhsnosnhosnbhsbhsinpbhosmcnhosnpbinbhsbinhosinposnbinhsnposcoscnosnpbinpbhoscosbhsnpbhoscosnosirhsnposinbinposnpbinposmnpbhsbhosinbhscnhoscscnhsinpbinhscnosbhosnbhsbhsninbhoscnpbininhosbinosnbhscnposbinhsbhosbinpbhsnhsbinbhosnhosnbhosnposinbhoscsbhscsninposnhscnbirninoscnhsnhosbinbinbhsbhscsmnhsinhosnoscnhoscscscosmninacnpbinosbhscninhsbhoscsnhosnposnposnposnposbhosbhsnpbinosinpbinpbhsbinhoscninpbhosbinbirnirnposnposnposnposbinposinosbhsnbininposnposnposinacoscnoscosnposnbhsnpbhsinhscosinposinposcosininosnhsinhosinbinhosbhsnposnhosbhoscnposcscnpbinpbhosninposcosbhosinhsinhscnpbinininpbinhsbinpbhsininpbhsinposcnosbhosbhosinosinhsmninosnhosinoscsininposnosinosininoscnasnhosbhsnhosmnhscnoscnhosbhosinhsinhscsbinposnbhsnposinposcnhsnbhscsnposnpbhsbinhosbhsnhsbhsnpbinininhscoscsnosinosbhsnhoscosninininposcnosnposcsnbhsbhsbhsinposnosbhoscninosnhscnposcnoscnosmninhscscosnhsnposcnhsinpbhsbhosmosnosinoscninpbhsmcnbhscosbinosbhsnbinposnbhsmninbhosinbhosmoscnposbirbhsbininposnhoscnosbhsnhsinposinosnpbhosninhsinhsbinbhsbhosbinosnpbhsbhscosnbhscnasnposnhosinininbinposnpbhsnhsbhoscsinosinpbhsbirnhsbhosbhscsnosmcnhosnoscnosninposirbininpbirninbinhosnpbinoscnosnpbhsnpbhscsnbinpbinhscnhsnbhsbhsbinpbinhsnpbhoscsbhoscnosbhoscnbinpbhscosinbinhsinosnbinbhsbinbinasnbhosnhosnosbhscsnosinhsbhsnhsmnpbhsnposininpbinoscnosinhsmnhoscoscsinbinhsbinbhsnasbhosbhosirhsninposbhosmcnhscsbinosininposninbinpbhosbinpbinbhscnpbinosinhsmcninhosmnposnosmnhsbinposnoscosinosbhosbinhsninosnhsinhsnbinpbhsirnosininhsnposnbhosnbhscscnposbinpbhoscsinosnposmcscnhsnbhsninosinhsbhsnhsnhoscnhsbhsirnhsnpbininpbhosinpbhscnposcscosnbhscscnininpbhsnbinininpbhscosinpbhosininhscnhsnosinhsnpbinhsbinacsbinposbininpbhoscsinpbinirhscninbhsnhsbhosinpbhscsbhscscnposbhsnosmnosbhosnhsnhoscscosbinhscnposbhsinbhscnpbhscnosbhsbhsinbhscsnbhosnhosnposcnacnposninposinbinposnbhsnosirbinhscnbhosnhoscsbhsbirbinhosmcsnhsbinposcninhsnbinhscninhsnpbhosbhsnhscnascnbhsnpbhsbhosnhscosinhsnhsninposninosmnhsinoscscscnbinhscnposinbhosmosnhosninbhosinbinpbhsnbhosininpbinbinacsbhoscnasinhsbhoscsnposnbhscnpbinhsininpbinpbhscnbhosbhsnosbhosnposcnasinposbhsbhscscosmnbhoscoscosmnosinposinhscnposcnpbinpbhsbhsninpbhsnpbinpbhosinhscnasinininbinininposbhsinhscosnbininbhsnbinbhsinosnpbhsnhsbinposnoscsmnbinosnhscnpbhscoscoscnpbhsninbhsnoscnhscosbhsnposcsbinbirninpbhosininosbhsmosnhsnbinpbinpbinhsinposinhsbhsnpbhsmninosbinbinhoscsbinbhsbhsbinoscosnpbhosbinirnpbinpbinbinbinbhosnbininhsinosbhscnhsnposnhsbinposmosnosbhosnposininbhosinposnbinhscsbinoscnosnbininhsbinposcscsnacsbinhoscnhsnposirnpbinposinbhsbinpbinhsmninpbininposbininbhosinpbinpbhsnbhosbinininbinbinhosbinosninhosbinpbhsnininhosnosbhscnpbinhsnininpbinbirhosinpbhsnposnpbirninbhosbinosnosbinacsninpbhoscnposcosnpbinbhsmninosbhscoscsnposbhosmcnposinposinposnposnbhsinirbinposcsinirbinposinininosbinposmninpbhscnbhsnhsinhsnhscsnposcosinhsinbhosnpbinpbhosnposinoscnposcnosnininoscsbhsninosbhosininininbinhsbhsinbinposbinposbhoscsbinposmosnposnosnbinhsnhsninhsmnbhosmcsmcnbhosbinhsinbhoscsnoscninpbinposcnirnpbinpbinhsnoscnhosnposinbinposbhosnbhoscscsninosnposnbhsinpbinpbinhsnposmosinosbhscscnoscninhsbhosinposbinbhsnbhosmnpbhsinposcosnposnhsmcscninpbinacsnpbinbinbinhoscsbhsinposmcnosinbhosinhsinhscnbirnhsbirnhoscosinoscnbhosnosbhsnpbhscnposcnbhscosnposininosbirnosinpbinhsnbinbinposnposbirhosnposmnhsinbhoscscsnhosbhoscnhsinhsbinhosininpbinhosnbirninhsbhosinhsbhscnhsirnosbinbhsbinpbhsnosinbhosnposbhosbinhosnbinhsnposnbhosnhsirhsinposninbinposininoscoscosnpbirninoscnasinhsmcscosinosinasinbirninposmnpbhsbinbinbinosmcsninbinpbhosbhsnosinpbhosnbhsnposnbinposinbhsnosinbhsnoscnasbininposininosinosinbhosnpbhscsinhosbhsnpbinininosbhoscsbhscosbhsnhsbinposbhosninosnirnposnhscosbhosnhosnposbhosmnosnpbinposbhosinbinhsnhsbhsnosbhosinoscnirbinpbhsninbinhosbinbhscsinhsbhosnposbinposinhscsnhosnpbininbhsbinbhsmnbhoscsnosnhscnasnpbinpbhsbinoscninhsnpbhscninposinosnirnpbinhsnbinbhsninbinhosbinposmcsinhsninbinposnhscnposmcnbhsbhosnosnosbhosinbhoscnasnbinposninbhsbhsirhsinhosbhsnposbinhosbhsnpbhosmosnposinbhsninosininhoscnhoscnascnpbinpbinbhosbhosnpbinosnhsmosbhsbinbhosbinosnpbhsnhsbhosinhsnpbhsmcninbinpbhosnbinhsbhosirbhsbhsnpbhsninpbhsbhsinosnhosbhsnpbinhsnpbinosirbhosininoscnpbhscnosininpbirhosbinpbhscnininoscnbhsbhsnpbhosnhsinposinasbhosbinosnhsnposbhsmnbinhsnposnbinbinposbinposbhsbinposcosbhosmcnposcnhscnasbhosmnhosnosmcoscnpbhsbinpbinposbhsbininosnpbhscsbhsmnhsnhosbirbinosninposmcsnhsnbinhsbhosninpbhosinhsbinhosinosnposcnhsnposnbhscsbhosinposnbirnpbhosnhosnhsnbinbirhoscsnposnbhsinpbhsnpbinposininoscnpbhosninosnposbhosinhosbhsninposbinpbhosmninhscscninhsmninposcnposmnhsnhscscscnbhsinosnpbhosnhoscosbinhscscnhosnasinosinhscosbhscnbinhosbinhscosnbirhsbhosnposnbhsininhsbinpbhoscoscoscscsbinasnbhosinpbhsmcnosnhsmcnhosinosnpbinpbhosmcosmninosnbinosnbhosbhosnosinbhosbhsnininposnhoscnbinosnposnbinhsbhsninosinbininhscsmoscnhscninacosbhoscnpbinbhosbinpbinbinposinposnpbinposcsbhsbirbhosinosmninosmcscosinbhosnpbhsmcnpbinoscnoscnasbhosbhscscsinpbhosinbhosinosbhscosbinpbhosinpbhoscscnirhosmnbhosinposcninoscnhsbinpbhsnpbhosbhosnhsninbhosnhosnposbhsnhsmnosinbinosnposnbhscnbinoscscninposnosbhsbhscoscscsninbininposininosirbhosnoscoscnposmosbhsinhosinbhosinhsnposbinpbhsbhsmcosinposcosinoscsnhsbinbhscnosninbinposbinbinbirnhsbinposinhosinhsnoscscninpbinininosnhosinhsnposcsirhscnacosninposbhscosnpbhsnhsinposcosinosinirbirnbinposmnposnhscsirnbininosnposcnasinosnhosirnpbhscoscnbhoscsnpbinpbhosnosinposnhsnhsnposinbhosninhsnbhsnbhosbhosbhoscnosnbhscscsbhsbinposnbinpbhsbinbhoscsmnpbhscscnbirhosmnhsnposnbhsinpbinhscsbhsninpbhscnbhscsinirnpbinoscosnosbhscninposcnoscnpbhscnoscsnposcosbirhsinbhosninposnbirhsnbirbinbhosbinpbhsmcscosnhsirbinhsnhscosinbinbinbinposbirhsnhscscscnacsninoscsinhosnbhsbininbhsnosinbinposbinpbhsbininbhscosinininosininbhosbhsnpbinosinhosnpbhscnasnhsinininposcsinosbhsnbhscninposinosnhoscninposnacsbhscsinpbhscnpbinbhosmnhsinhscnasbhoscnhsbirbhoscosinbirhosnirhsbhsnposnposbinhsnhscnininbhosinbinpbhscosnbhosninposnirnhoscnhsnhosinhscnacsnascnhsmnosbinposinhsbinposinposnhoscosnbhsbhscosnpbhsnpbhsbhsnoscosninhosnbhosnhosinirbhosbhsbhsbininosnbhosbinirninbhsnposbhoscosbhosbinposinosbhsninhsnasnosmcnhosnposmnhsnposbininhoscsbhsnasbhscsnbinosmnhsninhosnpbhoscnpbhscsbhsbhscnposcsinbhsinpbinposinhsinhosninbinpbhsbinpbinosinosbinbinpbinbhosnposcnosnbinbhscnbinoscnbhoscsinbinininbhsinhosnhsnpbhscosininpbhscsbinoscnposbhscosnpbinbininosbhsnosnbinoscsinpbhsbinosnbhscosnpbhoscnpbinosinosinposninosnhosinbinosnbhscosbhsnpbhsbhsbhscnpbhsbinposcsmnosnpbinpbhosbhsnosnbhsininosininpbhoscsbhosnposninirnposbhosbinposbinposnbhsirhosbinoscsmosbhscninhosnhsmnbhosbhosinasnposnpbhscosnposcnbhoscsnposcnasbhsnpbinpbinbhosinpbinpbhsinhscnasinosnposcnacscsbininbhsnhsnhsnbhsinpbhsninbhsbhscsinbhscsinposinhsbinposmnbhsininposnhosmnosnhsbhosnbininpbinpbirnhscnoscnpbhsinhscnosnpbinhscosnbhscsbhosininhsinpbinbhscnbhoscsnosmosnbhsinbinosnposcsinbhosinbinposcsnhsninosbhsmnposnposnpbhsnhsininbhosinoscoscnposbinbhsnhosbhosnpbhosbhsmcosinposcnosinininosinoscsmcsnosinbinbininosbinhosnposcnposcsnpbhsinpbhsinosinpbinposnhsbinhsnpbinascsmoscnposinhsnposcnosnpbinbininbhosbhosinbinposbhsnacosinoscosbinosmcosinbhsnbhsnasinosbhosmcnhsbhsininpbinoscnhscnpbhscnacnposnposcnposnposnposnpbinbhsnasnposininbinpbhosbirbinosbhsbhsnbininbhsmnosnhsbinpbhsnhsnhsinpbhsinosbinbinposbhscsnascsinbinpbhsbinhsnbinosmnhscsinbhsnbhsnpbhosininbinosmnhsbhsnbhsinbinoscosnbinposcsnposnhsnpbinpbhscnbinhoscnposnascninhsinposnosinosnposcosnhoscoscnbhosbhosininpbhsnposbhsbinirhsinosinoscsmnhosinosinposcnosinhsbhosnposnosnbhosinpbininoscnpbinininbhoscnascnhsbinposninbhsnposbhosnpbhosnposininpbinhosninbhsbirhscnhscscnhosninosinosninposbhosmosninhsbhsnhsinpbhsnposbinacnasnposinhsbhosbhsninhsnpbhosnpbhosinosbhosnhscnhscsmnposmnpbhosbininhsininosbhoscnpbhosinhsbinhsnposbhosmosnhsbhosinhsnhscnposcninhosninbhosnhosmnpbinininhscnininbhosnosnacscsninhscnpbhscnposnpbirbhoscnacsbhsinpbhsninbinoscnbinbinposnbirninposnbinhsnbhscnhoscnhsinhsnposnbhsbhoscscsninirninposnpbhsbinposcnpbhscsinininbinhscsinpbhscosnbinhsbhosmnbirhosinhscscscscnbhsnosininbinosnosnpbhsnasnbinposnpbinosmcosnposininbinpbinposcnosbhscnbhsmnbhosinosnposbhsininhosnhscnpbinosbinposbhsmoscosbhosinoscnbhosnbinbhsbinpbhosbhosirnposninosnbhoscosinpbinbhosnposbhsnposcsmnbininininoscnpbinhscnacnbhsinhsnposnposnacsinbhscosnposbhosnosbhsbhsinoscnosninhsinoscsinhsnposcsbininhosnhscsmcnacosbinposcsbhosmnininhsbhsnposninpbhsnposininhsnoscscscsnhsbhsnosbhscnbhsinbinbhsnposnpbhosnbhscnhsnhosnhscosmnpbininosbhosbinposbhsinposbhosnbinbinbinpbinbhsnbhsinhosnhscnbhosnbhosbinhosbhscsnbhosinosbhosmcnosinpbinosbinhsnpbirninpbhsbhsninosnbinasinhsmnbinposnpbinosininpbinpbhosinhsnhosinoscninhscnposcoscscnbinpbhsnininpbhsmcsbinpbhsinacsinbininhsinposnbinhosinpbinhsnposnasbinhscnhosnoscsinbhsnhscnascnosnhscoscsbinbhscnbinbhsbinpbinhscsbhsbhsbinosnhosinpbinosinoscninbhosinbhsnbirbhsinbhosbhsinposnhscnposinhsbhsnpbhsnpbhsbinirnbinosininosinbhosbinpbinbhsinbinpbhosnposmcnpbhosininhsinhosnbhoscninpbinpbinhsnbhosninbinirhosnposnhsbhscnbininpbhosmnpbinhscsbhsnosinhsbininbinpbininpbinosinoscsbhscosnosnhoscosinbhsnininosninbhsbhoscsbhscscosnposinhsninposininhosnhosbinhsnoscosnininhosnposmcsbhsmnhsininbhsnhsnposbhosbinbhsbinhosininbhsbinininhoscscnposnbinpbhsnbhscsinhosnhscnpbhsbhoscnasbhsbinpbinbinbinhsnhsinposmcnposnininbhsnbininbinhosmcnpbhsnosnposinhsnosnosinhsbhosinposnhsininhsninosbhsinpbhsininpbhsinhsinhsnosirhoscsinasbirnhsnpbhsnpbinhsnbhsnhsbhscnosmninbinpbhsnhsnposbhosnpbhsbininininbinhscninosnpbinosnpbirhsnpbhsbhsinhsnhosnpbhsbhsinposnasinhosnpbhsnbhosnosnosinosbinbhsnposcninhsbhosbhosinposbinpbininininhsinposcosnposnpbhosnposcscsnposinposbhsbhosinpbhsinbhosinposnosnbininposcoscsnbhsnhosbhsbinbhosnpbinhsbhosbininoscnasinpbinirhsnpbinbinpbhsmcoscnininosnhscosnbhsnbinpbinhsinpbirbhsininpbhsbhosnpbinhsinbirbhosbinhoscosnposininbinposnposmnbhosnpbhsbhsnposcsinbinpbhosininpbhsnposbhsnposbhsninposninirnosinbirhsnbhsnosinpbhsinoscnhsinosnbhsininasbhsnbinposcsinosinosmnbinosinpbhosbininposnhosbhsbinpbhscsmnhsninbhsbinhsnosnpbhsinbinhsmcnpbirhosbinhosbhosinosinosnposbhscosbhosnposbhoscnhsbhosnhsbinpbinosbhsnposbhsinhsbinbhsmnoscsnhsbinposcsnpbinbhoscninosmcosnosnosinosnpbhsninhsnbinpbinhosnosninbinhscsbhosinosmcnpbhoscninpbinoscosnbinposinhsbhoscsnbinbhsbinhsmnpbhscsinpbininasinpbinasbhsbhsbinpbininposcnh"

@test.describe('group 1')
def tests():
    @test.it("check memory crash")
    def _():
        @test.timeout(8)
        def _():
            test.pass_()
            XXX(DATA)    # this explodes the heap
            test.fail("didn't fail...")
