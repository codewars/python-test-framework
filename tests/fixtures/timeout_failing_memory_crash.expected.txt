
<DESCRIBE::>group 1

<IT::>check memory crash

<FAILED::>Something went wrong: the process running the function crashed without feedback (probably saturating the available memory)

<COMPLETEDIN::>30.41
